src/rpc/api/wire/types.ts

```diff
@@ -0,0 +1,10 @@
+import {Slot} from "../../types/primitive"
+
+export interface BlockRootSlot {
+  block_root: Buffer;
+  slot: Slot;
+}
+
+export interface HashTreeRoot {
```

Comment:
I think we could make a `Root` type in `src/types/primitive.ts` that would be useful across the codebase, used in the wire messages.

It would look like:
```typescript
export type Root = bytes32;
```
and since we would be using it in ssz-ed types, we'd need a `export const Root = bytes32;` too.